typings/index.d.ts

```diff
@@ -238,11 +303,12 @@ export class BitField<S extends string, N extends number | bigint = number> {
   public toJSON(): N extends number ? number : string;
   public valueOf(): N;
   public [Symbol.iterator](): IterableIterator<S>;
-  public static FLAGS: unknown;
+  public static FLAGS: Record<string, number | BigInt>;
```

Comment:
```suggestion
  public static FLAGS: Record<string, number | bigint>;
```

Reply:
Shouldn't this be `Record<S, bigint>` instead?

Reply:
This is BitField, which can be either.

Reply:
Disregarding that bigint part, the key is still relevant

Reply:
`FLAGS` is static so you can't use the type parameter 😭